src/end_entity.rs

```diff
@@ -164,6 +195,7 @@ impl<'a> EndEntityCert<'a> {
     /// `cert` is the purported end-entity certificate of the client. `time` is
     /// the time for which the validation is effective (usually the current
     /// time).
+    #[deprecated(since = "0.101.2", note = "Use `verify_for_usage` instead")]
```

Comment:
also deprecate `TlsServerTrustAnchors`, `TlsClientTrustAnchors`?

remove rather than deprecate `NonTlsTrustAnchors` and `verify_is_valid_cert_with_eku`? (they haven't been in a release yet)

Reply:
Good call.